sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/SymbolicExecution/ExplodedGraphTests.cs

```diff
@@ -265,47 +184,27 @@ public void ExplodedGraph_SingleBranchVisited_And()
                     }
                 };
 
-            explodedGraph.Walk();
-
-            explorationEnded.Should().BeTrue();
-            numberOfExitBlockReached.Should().Be(1);
+            context.Walk(3); // Only EG final status check
```

Comment:
Why is this comment necessary here ? Isn't it the case for all (except 1) tests ?